parser/jvm/src/test/resources/txt/bank0.txt

```diff
@@ -0,0 +1,201 @@
+1|Bank0_RID|7|201803071050|2017|99-1234567|200|bank-0 National Association|1650 Biscayne Blvd |Miami|FL|33132     ||||||Jane Smith|111-111-1111|222-222-2222|jane_smith@bank0.com
+2|Bank0_RID|7|8299422144|20170220|1|1|1|2|78|3|1|20170226|16974|17|063|0001.03|2|5|5| | | | |8| | | | |2|5|173|6| | | |NA   |2|1

```

Comment:
Until [this PR](https://github.com/cfpb/hmda-platform/pull/883) is merged in, you'll want to respondent ID of the LAR to match the `id_rssd` of the bank you're testing.  In this case, `6999998`.

Currently, this will cause a 500 error on the IRS endpoint and a blank response on the modified LAR endpoint.

The same thing will need to be done for `bank1.txt`.  Or wait until the other PR is merged in, after which any LAR file will work.